docs/github.md

```diff
@@ -91,7 +92,23 @@ $ jj commit -m 'address pr comments'
 $ # Update the branch to point to the new commit.
 $ jj branch set your-feature -r @-
 $ # Push it to your remote
-$ jj git push.
+$ jj git push
+```
+
+Notably, the above workflow creates a new commit for you. The same can be 
+achieved without creating a new commit. 
+
+```shell script
+$ # Create a new commit on top of the `your-feature` branch from above.
+$ jj new your-feature
+$ # Address the comments, by updating the code
+$ jj diff
+$ # Give the fix a description and create a new working-copy on top.
+$ jj describe -m 'address pr comments'
+$ # Update the branch to point to the current commit.
+$ jj branch set your-feature -r @
+$ # Push it to your remote
+$ jj git push
```

Comment:
@joyously Does this address your concern [here]?

[here]: https://github.com/martinvonz/jj/issues/1989#issuecomment-1693723411

Reply:
Yes, thank you, although on the page I'm writing this, GitHub says "Outdated" on this, but I couldn't find your comment otherwise. Perhaps that's more to explain in this doc?

Reply:
It's outdated, as the comment in this version is factually incorrect. We do not create a new working-copy on top. I immediately corrected it and re-pushed the commit. This comment is attached to a old version of the explanation.